internal/pbio/internal/test/thetest.proto

```diff
@@ -0,0 +1,50 @@
+// Protocol Buffers for Go with Gadgets
```

Comment:
Content of this file is (partially) copy-pasted from gogo/test/thetest.proto

(only message that is needed to test pbio)